pkg/registry/apis/secret/contracts/outbox_queue.go

```diff
@@ -19,7 +19,7 @@ type AppendOutboxMessage struct {
 	Name            string
 	Namespace       string
 	EncryptedSecret secretv0alpha1.ExposedSecureValue
-	KeeperName      string
+	KeeperName      *string
```

Comment:
Keeper everywhere is optional, maybe looks a bit meh but we rarely needs to dereference it from what I've seen, it's mostly passing it around as-is.